### PR TITLE
Add entries from dynamic and hybrid reading group.

### PR DESCRIPTION
**Pull request by namcsi**

### CODE REVIEW

File: krr.bib [hunk 1]
```diff
@@ -5689,6 +5689,16 @@ @inproceedings{cafascwa20b
   pages = {193-202}
 }
 
+@article{cafascwa21,
```
Comment (tortinator) [inline krr.bib:5692]: This lacks a character at the end - the other question is why insert the arxiv now that the journal article has been published..? I'd rather insert the journal article!

File: krr.bib [hunk 2]
```diff
@@ -18685,6 +18695,20 @@ @inproceedings{nisiso99a
   year = {1999}
 }
 
+@inproceedings{nivpit10a,
+  title = {From MTL to Deterministic Timed Automata},
```
Comment (tortinator) [inline krr.bib:18699]:
```suggestion
  title = {From {MTL} to Deterministic Timed Automata},
```

File: krr.bib [hunk 3]
```diff
@@ -18685,6 +18695,20 @@ @inproceedings{nisiso99a
...
+  author = {D. Ni\v{c}kovi\'c and N. Piterman},
+  editor = {K. Chatterjee and T. Henzinger},
+  address = {Berlin, Heidelberg},
+  booktitle = {Proceedings of the Eigth International Conference on Formal Modeling and Analysis of Timed Systems (FORMATS'10)},
```
Comment (tortinator) [inline krr.bib:18703]:
```suggestion
  booktitle = {Proceedings of the Eighth International Conference on Formal Modeling and Analysis of Timed Systems (FORMATS'10)},
```

File: krr.bib [hunk 4]
```diff
@@ -18685,6 +18695,20 @@ @inproceedings{nisiso99a
   year = {1999}
 }
 
+@inproceedings{nivpit10a,
+  title = {From MTL to Deterministic Timed Automata},
+  author = {D. Ni\v{c}kovi\'c and N. Piterman},
+  editor = {K. Chatterjee and T. Henzinger},
+  address = {Berlin, Heidelberg},
```
Comment (tortinator) [inline krr.bib:18702]:
```suggestion
```

File: krr.bib [hunk 5]
```diff
@@ -18685,6 +18695,20 @@ @inproceedings{nisiso99a
...
+  booktitle = {Proceedings of the Eigth International Conference on Formal Modeling and Analysis of Timed Systems (FORMATS'10)},
+  doi = {10.1007/978-3-642-15297-9_13},
+  keywords = {Model Check, Controller Synthesis, Time Automaton, Deterministic Automaton, Clock Constraint},
```
Comment (tortinator) [inline krr.bib:18705]:
```suggestion
```

File: krr.bib [hunk 6]
```diff
@@ -24605,6 +24629,15 @@ @article{wolper83a
   year = {1983}
 }
 
+@article{wolper85a,
+  title = {The Tableau Method for Temporal Logic: An Overview},
+  author = {Pierre Wolper},
+  journal = {Logique Et Analyse},
```
Comment (tortinator) [inline krr.bib:24635]:
```suggestion
  journal = {Logique et Analyse},
```
???

File: krr.bib [hunk 7]
```diff
@@ -24605,6 +24629,15 @@ @article{wolper83a
   year = {1983}
 }
 
+@article{wolper85a,
+  title = {The Tableau Method for Temporal Logic: An Overview},
+  author = {Pierre Wolper},
```
Comment (tortinator) [inline krr.bib:24634]:
```suggestion
  author = {P. Wolper},
```

**Review by tortinator — CHANGES_REQUESTED** [review]
Thank you, @namcsi, though some work remains :innocent:

File: krr.bib (hunk 1 above)
Comment (namcsi) [reply krr.bib:5692]: Gotcha. I didn't find any other bib than this arxiv one after doing some browsing. Which journal was it published in?

File: krr.bib (hunk 4 above)
Comment (namcsi) [reply krr.bib]: Fixed and rebased!

File: krr.bib (hunk 2 above)
Comment (namcsi) [reply krr.bib]: Fixed and rebased!

File: krr.bib (hunk 6 above)
Comment (namcsi) [reply krr.bib]: Don't speak French;) fixed!

File: krr.bib (hunk 7 above)
Comment (namcsi) [reply krr.bib]: whoops, fixed!

File: krr.bib (hunk 3 above)
Comment (namcsi) [reply krr.bib]: Fixed and rebased!

File: krr.bib (hunk 5 above)
Comment (namcsi) [reply krr.bib]: So keywords should not be included in general? I saw a couple entries with a keywords field and was unsure of the policy. Fixed!

File: krr.bib (hunk 1 above)
Comment (tortinator) [reply krr.bib]: [algorithms-v16-i04_20230503-bib.txt](https://github.com/krr-up/bibliography/files/11384751/algorithms-v16-i04_20230503-bib.txt)

**Pull request merged**